src/Fantomas/Program.fs

```diff
@@ -35,12 +37,13 @@ type Arguments =
                 sprintf
                     "Input paths: can be multiple folders or files with %s extension."
                     (Seq.map (fun s -> "*" + s) extensions |> String.concat ",")
+            | Verbosity _ -> "Set the verbosity level. Allowed values are n[ormal] and d[etailed]."
 
 let time f =
     let sw = Diagnostics.Stopwatch.StartNew()
     let res = f ()
     sw.Stop()
-    printfn "Time taken: %O s" sw.Elapsed
+    sprintf "Time taken: %O s" sw.Elapsed |> stdlog
```

Comment:
I guess all the `stdlog` calls can be interpolated strings instead.
The extra pipe seems redundant.